### PR TITLE
Tag 38

### PR DESCRIPTION
**Pull request by KedarSki**
Tests performed

### CODE REVIEW

File: TagsterWebAPI/TagsterWebAPI.UnitTests/TagsControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using TagsterWebAPI.Controllers;
+using Xunit;
+
+namespace TagsterWebAPI.UnitTests
+{
+   public class TagsControllerTests
+    {
+        [Fact]
+        public void InsertTagsData()
```
Comment (flapek) [inline TagsterWebAPI/TagsterWebAPI.UnitTests/TagsControllerTests.cs:9]: There is nothing checking. If class is unnecessary, delete it

File: TagsterWebAPI/librariesTests/Tagster.Exception.UnitTests/Services/CookieFactoryTests.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tagster.Infrastructure.Services;
+
+namespace Tagster.Exception.UnitTests.Services
+{
+    public class CookieFactoryTests
```
Comment (flapek) [inline TagsterWebAPI/librariesTests/Tagster.Exception.UnitTests/Services/CookieFactoryTests.cs:10]: There is nothing checking. If class is unnecessary, delete it

**Review by flapek — CHANGES_REQUESTED** [review]

File: TagsterWebAPI/TagsterWebAPI.UnitTests/TagsControllerTests.cs (hunk 1 above)
Comment (KedarSki) [reply TagsterWebAPI/TagsterWebAPI.UnitTests/TagsControllerTests.cs:9]: This cannot be found after sync. I assume it should be resolved now

**Comment by KedarSki** [comment]
All should be resolved now

**Review by flapek — APPROVED** [review]

**Pull request merged**